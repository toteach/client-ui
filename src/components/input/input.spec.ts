import { mount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TInput from './input.vue';

describe('TInput', () => {
  let wrapper: Wrapper<Vue>;
  beforeEach(() => {
    wrapper = mount(TInput, {
    });
  });
  it('is initialized', () => {
    expect(wrapper.name()).toBe('TInput');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
