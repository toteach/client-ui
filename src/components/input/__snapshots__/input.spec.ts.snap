// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TInput renders correctly 1`] = `
"<div class=\\"t-control t-control_type_text\\"><input type=\\"text\\" class=\\"t-control__input\\">
  <!---->
</div>"
`;
